src/components/TopNavbar/TopNavbar.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="left-0 top-0 bg-white fixed w-full rounded">
+    <div class="z-30 relative max-w-screen-xl w-full m-0 m-auto py-0 px-6">
+      <div
+        data-collapse="medium"
+        data-animation="default"
+        data-duration="400"
+        role="banner"
```

Comment:
We also probably want to remove this role. This isn't a banner... It's more like nav